src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+      Lottery ball1 = new Lottery();
+      ball1.setNumber(ball1.getRandomBall());
+        System.out.println("ball number:" + ball1.getNumber() + "     " + "ball color:" + ball1.getRandomColor());
```

Comment:
The 'System.out.println' statement is overly complex and hard to read. Consider storing the ball's number and color in local variables before printing them to simplify the code.